dateutil/rrule.py

```diff
@@ -707,7 +707,7 @@ def __str__(self):
             parts.append('INTERVAL=' + str(self._interval))
 
         if self._wkst:
-            parts.append('WKST=' + str(self._wkst))
+            parts.append('WKST=' + repr(weekday(self._wkst))[0:2])
```

Comment:
I don't think it's possible to call `MO(0)`:
```python
>>> dateutil.rrule.MO(0)
Traceback (most recent call last):
    ...
ValueError: Can't create weekday with n == 0
```

That said, `str(weekday(self._wkst))` appears to produce the right value already. I only did `repr(day)[0:2]` here because that was the pattern used further down in the file, and I figured I'd follow suit in case there was something more complex going on that I didn't understand.